schedulezoomtalks.py

```diff
@@ -281,8 +281,10 @@ def schedule_talks(repo, talks) -> int:
         if meeting_id:
             talk["zoom_meeting_id"] = meeting_id
             talk["registration_url"] = registration_url
+
             # Add this talk to researchseminars.org
-            # publish_to_researchseminars(talk)
+            publish_to_researchseminars(talk)
```

Comment:
This isn't mission-critical, but if it fails now, we won't notify the speaker. Therefore we should either do try/catch, or move it to the end.

Reply:
Moved it to the end for now. We will be able to see when this fails during testing now if they don't show up as approval requests.

Reply:
Sorry, that's not far enough. Now if this fails, the bot won't update the talks.yml.

I propose try/catch + `logger.error(...)`.